### PR TITLE
Update branding guidelines

### PR DESCRIPTION
**Pull request by dankohn**
Signed-off-by: Dan Kohn <dan@dankohn.com>

This change has been recommended by Linux Foundation legal counsel. It brings Kubernetes logo usage guidelines into alignment with other logos and trademarks managed by the Linux Foundation.

The CNCF governing board has requested the change while also voting to add an additional permission not normally granted: "Additionally, permission is granted to modify the Kubernetes mark for non-commercial uses such as t-shirts and stickers." Importantly, this change removes permission for Certified Kubernetes offerings to *modify* the Kubernetes logo. [None](https://landscape.cncf.io/category=certified-kubernetes-distribution,certified-kubernetes-hosted,certified-kubernetes-installer&format=card-mode&grouping=no) of the 97 certified products currently do so, but we don't want them to start.

The Kubernetes Steering Committee should sign off on this.

Cc Kubernetes Steering Committee: @spiffxp @dims @timothysc @derekwaynecarr @michelleN @pwittrock @joebeda @brendandburns @smarterclayton @bgrant0607 @sarahnovotny @philips
Cc Original author: @thockin 

/committee steering
/kind documentation

### CODE REVIEW

**Comment by dankohn** [comment]
Cc LF legal counsel: @swinslow

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 
/hold 

Waiting for standard vote ^ 

@dankohn  - is there a tracking issue in the steering repo that references this PR?

**Comment by dankohn** [comment]
@timothysc there is no tracking issue. You suggested offline that one is not necessary.

**Comment by philips** [comment]
Lets reset the lazy consensus timer on this. An email should have gone out to steering@kubernetes.io first to get everyone's attention.

File: logo/usage_guidelines.md
Comment (dims) [inline logo/usage_guidelines.md]: we may want to remove this line `You
can use the logo on your website or in print without pre-approval, provided you
follow these basic guidelines.`

Comment (dims) [reply logo/usage_guidelines.md]: as it is covered below at the end

**Review by thockin — COMMENTED** [review]
/lgtm

**Comment by bgrant0607** [comment]
LGTM

**Comment by dims** [comment]
LGTM

**Comment by timothysc** [comment]
LGTM

**Comment by timothysc** [comment]
Timeout... 
/hold cancel

**Comment by timothysc** [comment]
Following lazy consensus rules here.

**Pull request merged**
